### PR TITLE
HDDS-10829. Suppress extra INFO log lines from ozone shell commands output

### PR DESCRIPTION
**Pull request by sarvekshayr**
## What changes were proposed in this pull request?
Changed the log level of the extra log lines from INFO to DEBUG. 

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-10829

## How was this patch tested?
This patch was tested by running the ozone shell commands on docker. 
(disabled the function `ozone_suppress_shell_log` for testing purpose)
Before suppressing the logs:
Screenshot 2024-05-08 at 4 17 15 PM

After suppressing the logs:
Screenshot 2024-05-08 at 4 03 20 PM

### CODE REVIEW

**Review by duongkame — APPROVED** [review]
Thanks for the patch. +1. @sarvekshayr

File: hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/ClientTrustManager.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ private void initialize(List<X509Certificate> caCerts)
   private List<X509Certificate> loadCerts(CACertificateProvider caCertsProvider)
       throws CertificateException {
     try {
-      LOG.info("Loading certificates for client.");
+      LOG.debug("Loading certificates for client.");
```
Comment (adoroszlai) [inline hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/ClientTrustManager.java:148]: Looks like integration test is looking for this log message:

```
Error:  Tests run: 6, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 11.809 s <<< FAILURE! - in org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS
Error:  org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.testLongLivingClientWithCertRenews  Time elapsed: 0.857 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Client loaded certificates. ==> expected: <true> but was: <false>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)
	at org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.assertClientTrustManagerLoading(TestOzoneContainerWithTLS.java:288)
	at org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.testLongLivingClientWithCertRenews(TestOzoneContainerWithTLS.java:229)
```

Comment (adoroszlai) [reply hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/ClientTrustManager.java:148]: @sarvekshayr `TestOzoneContainerWithTLS` needs to set log level for `ClientTrustManager` to `DEBUG`

**Comment by duongkame** [comment]
Thanks @sarvekshayr for the patch and @adoroszlai  for reviewing.

**Pull request merged**